### DISCUSSION TITLE
Final variants file missing

### DISCUSSION THREAD

**Discussion by Florencia-DeLillo** [PGScatalog/pgsc_calc#114] [Q&A] [answered]
Hi!! I am having trouble finding a file. Does anyone know how can I access the file containing the variants used by the "PGSC_calc" pipeline for generating the PRS scoring file? If I am right,  only a subset of the original scores is employed in the final PRS calculation.  Could you please tell me where I can find that file please?

**Comment by nebfield** [comment, marked as answer]
In `v1.3.2` you can find a log that describes the status of every scoring file variant in `results/match`:

For example, on the test profile:

```
results
├── match
│   ├── cineca-synthetic-subset_22_additive_0.scorefile.gz
│   ├── cineca-synthetic-subset_log.csv.gz <---- the file you want 
│   ├── cineca-synthetic-subset_summary.csv
│   └── versions.yml
├── pipeline_info
└── score
```

(your copy of the log will have a different file name)

See here for more information about the log structure: https://pgsc-calc.readthedocs.io/en/latest/output.html#match
